README.md

```diff
@@ -37,6 +37,48 @@ module "mlops_aws_infrastructure" {
 }
 ```
 
+### Usage example with MLOps AWS Project Module
```

Comment:
Should we link the project module docs here? In case the reader doesn't know what it is

Reply:
Oh I see we mention the infra module already in https://github.com/databricks/terraform-databricks-mlops-aws-project/blob/8db2db60ef0f344d38e6f67d7d5d00f8cbd28c16/README.md?plain=1#L12 - maybe we can link to the infra module docs there, for visibility